README.md

```diff
@@ -15,6 +15,7 @@ Branch     |[GitHub Actions](https://github.com/tresinformal/game/actions)
 `theo`     |![check](https://github.com/tresinformal/game/workflows/check/badge.svg?branch=theo)     |[![codecov.io](https://codecov.io/github/tresinformal/game/coverage.svg?branch=theo)](https://codecov.io/github/tresinformal/game/branch/theo)
 `tianjian`|![check](https://github.com/tresinformal/game/workflows/check/badge.svg?branch=tianjian)   |[![codecov.io](https://codecov.io/github/tresinformal/game/coverage.svg?branch=tianjian)](https://codecov.io/github/tresinformal/game/branch/tianjian)
 `yang`|![check](https://github.com/tresinformal/game/workflows/check/badge.svg?branch=yang)   |[![codecov.io](https://codecov.io/github/tresinformal/game/coverage.svg?branch=yang)](https://codecov.io/github/tresinformal/game/branch/yang)
+`Ludvig`|![check](https://github.com/tresinformal/game/workflows/check/badge.svg?branch=Ludvig)   |[![codecov.io](https://codecov.io/github/tresinformal/game/coverage.svg?branch=Ludvig)](https://codecov.io/github/tresinformal/game/branch/Ludvig) 
 
```

Comment:
Yup, that should work! Note that you are the first one that has a branch name that starts with a capital letter. I encourage you to blend in with the naming style of the team of the future, but not hard enough to request this as a change. Well done!